define/jsonhelper.go

```diff
@@ -0,0 +1,389 @@
+package define
```

Comment:
defines the encapsulating and structure of the message received through the websockets.
Also defines the functions to extract data from the byte array received. (Analyse**() )

Reply:
The global flow of messages received interleaves calls to handle**() and Analyse**() with the idea that Analyse interpret the json and then handle dispatch tasks accordingly and so on until we fully "unpeeled" the json layers.